src/components/WindowSideBarButtons.js

```diff
@@ -125,7 +124,15 @@ export class WindowSideBarButtons extends Component {
             )}
           />
         )}
-        <PluginHook TabButton={TabButton} PluginComponents={PluginComponents} />
+        { PluginComponents
```

Comment:
I like where this is going. Do we then have a new type of plugin here? One where a mode is not specified but can be extended into this area?

Reply:
I don't know if it's a new type as far as its consumers go. It's just like `add`, only... well, it works in  `Tabs`, which reaches into is children to pluck out the `value` prop.

Reply:
Ok.. I've updated the wiki accordingly.